docs/dev-guide.md

```diff
@@ -1,43 +1,49 @@
 ## Getting Mainflux
 
-Mainflux can be fetched from the official [Mainflux GitHub repository](https://github.com/Mainflux/mainflux):
+Mainflux source can be found in the official [Mainflux GitHub repository](https://github.com/Mainflux/mainflux). You should fork this repo in order to make changes to the project. The forked version of the repo should be cloned using the following:
```

Comment:
Even if it's understandable you can maybe use `repository` instead of `repo`

Reply:
This `repo` does not bother me at all, on the cotrarly sonds richer to me.

Reply:
@drasko maybe the first time in the document `repository` should be used and then `repo` form then on out.  Just to be completely clear that `repo` is referring to `repository`.

Reply:
Sounds good.